src/quantity_api/__init__.py

```diff
@@ -19,3 +19,22 @@ def unit(self) -> U: ...
     ### Dunder Methods
 
     def __eq__[B](self, other: Self[op.CanEq[V, B], U], /) -> B: ...
+
+    def __quantity_namespace__(
+        self: quantity, /, *, api_version: Optional[str] = None
```

Comment:
I think best to just omit the type of self here, or else use `typing.Self` which is made for this purpose.

```suggestion
        self, /, *, api_version: Optional[str] = None
```